walletnode/services/artworksearch/thumbnail/thumbnail.go

```diff
@@ -0,0 +1,163 @@
+package thumbnail
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/pastelnetwork/gonode/common/log"
+	"github.com/pastelnetwork/gonode/pastel"
+	nodeClient "github.com/pastelnetwork/gonode/walletnode/node"
+	"github.com/pastelnetwork/gonode/walletnode/services/artworkregister/node"
```

Comment:
you refer to same client interface to artwork_register to connect to supernode. Make me confusing.
Suggest define own gprc proto for artwork_search service in ./proto/walletnode. Even it's empty now.

Reply:
done